packages/shared/routes/dashboard/wallet/views/AccountHistory.svelte

```diff
@@ -25,7 +25,10 @@
     export let transactions: AccountMessage[] = []
```

Comment:
This isn't a part of your changes, but we currently mix the concept of "transactions" and "messages" :sweat:. We should keep this in mind while we're transitioning to the new wallet-rs changes that revolving around UTXOs and not protocol messages anymore (hopefully should be simpler for us).

Reply:
Yeah, I am on my third branch of nested changes and been refactoring stuff in this one too. For now I have changed the use of transactions with messages here. But its in a third PR ;) and will likely be refactored when we do wallet.rs changes.